.pre-commit-config.yaml

```
@@ -15,16 +15,17 @@ repos:
           - --fix=lf
         exclude: make.cmd
       - id: trailing-whitespace
+        exclude: \.md$
 
   - repo: https://github.com/asottile/pyupgrade
-    rev: v3.19.1
+    rev: v3.20.0
     hooks:
     -   id: pyupgrade
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)